package.json

```diff
@@ -71,6 +71,7 @@
     "@babel/preset-react": "^7.16.0",
     "@svgr/cli": "^8.1.0",
     "babel-loader": "^8.2.3",
+    "prettier": "3.5.2",
```

Comment:
Was prettier required to get it working in vscode?  -- are you using vs code or something else?
I don't want to have prettier as a dependency if we don't have to.  But if that was required then lets do it.  But can you uninstall prettier and see if it's working in your editor?   
Also, if it is required it should be in optionalDependencies.  You have to put it there manually in the editor.  When we push to production on heroku, devDependencies are installed and build and used to build the repo.  optionalDependencies are not installed in order to save space and build time.